src/main/java/decodes/tsdb/CpCompDependsUpdater.java

```diff
@@ -1000,18 +986,29 @@ public void evalComp(DbComputation comp)
 			}
 			if (!doingFullEval)
 			{
-				try { writeToAdd2Db(comp.getId()); }
-				catch(DbIoException ex) { /* do nothing -- err msg already logged. */ }
+				try
+				{
+					writeToAdd2Db(comp.getId());
+				}
+				catch(DbIoException ex)
+				{
+					/* do nothing -- err msg already logged. */
```

Comment:
except we are doing something?

Reply:
To be fair to myself, that was already in place and wasn't entirely sure if I wanted to deal with replacing it right now.
Probably good to at least log a warning message though in case it *wasn't* actually already logged.